dll/win32/shell32/dialogs/dialogs.cpp

```diff
@@ -1195,7 +1238,7 @@ static BOOL IsFriendlyUIActive(VOID)
                             (LPBYTE)&dwValue,
                             &dwSize);
 
-    if (lRet == ERROR_SUCCESS && dwType == REG_DWORD && dwValue == 0x1)
+    if (lRet == ERROR_SUCCESS && dwType == REG_DWORD && dwValue != 0x0)
```

Comment:
You've provided a defined macro, `IS_PRODUCT_VERSION_WORKSTATION`, below the code but about this `0x0` value?